pkg/cache/v3/experiments.go

```diff
@@ -24,6 +24,10 @@ import (
 	experimentproto "github.com/bucketeer-io/bucketeer/proto/experiment"
 )
 
+const (
+	experimentsTTL = 0
+)
+
 type ExperimentsCache interface {
```

Comment:
Since we are not using the `ExperimentsCache`, we can delete it as well.

Reply:
Thankyou, I deleted it 👍